docs/docs/addons-perf.md

```diff
@@ -32,7 +36,7 @@ See these articles by the [Benchling Engineering Team](http://benchling.engineer
 
 If you're benchmarking or seeing performance problems in your React apps, make sure you're testing with the [minified production build](/react/downloads.html). The development build includes extra warnings that are helpful when building your apps, but it is slower due to the extra bookkeeping it does.
 
-However, the perf tools described on this page only work when using the development build of React. Therefore, the profiler only serves to indicate the _relatively_ expensive parts of your app.
+However, the perf tools described on this page only work when using the development build of React. If you set NODE_ENV to _production_ than it will not work.  Therefore, the profiler only serves to indicate the _relatively_ expensive parts of your app.
```

Comment:
NODE_ENV is only applicable when compiling yourself - you might be using a CDN version. So I think this sentence is better just left out, the right way to refer to this is "using the development build" rather than "setting NODE_ENV".